go.mod

```diff
@@ -154,4 +155,6 @@ replace github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 
 replace github.com/cosmos/cosmos-sdk/db => ./db
 
+replace github.com/gogo/gateway => github.com/notional-labs/gateway v1.1.1-0.20220417180718-8e60e17a098d
```

Comment:
why this replace?

Reply:
pbjson typing

Reply:
sorry what I mean is:

this one was **very** important, it keeps the interface registry from exploding.

Reply:
What do you mean? Can you link an issue? we specifically want to avoid `replace`.